tests/test_self.py

```diff
@@ -341,6 +341,37 @@ def test_wrong_import_position_when_others_disabled(self) -> None:
             actual_output = actual_output[actual_output.find("\n") :]
         assert self._clean_paths(expected_output.strip()) == actual_output.strip()
 
+    def test_progress_reporting(self) -> None:
+        module1 = join(HERE, "regrtest_data", "import_something.py")
+        module2 = join(HERE, "regrtest_data", "wrong_import_position.py")
+        args = [
+            module2,
+            module1,
+            "--disable=all",
+            "--enable=wrong-import-position",
+            "--verbose",
+            "-rn",
+            "-sn",
+        ]
+        out = StringIO()
+        self._run_pylint(args, out=out)
+        actual_output = self._clean_paths(out.getvalue().strip())
+
+        expected_output = textwrap.dedent(
+            f"""
+        Using config file pylint/testutils/testing_pylintrc
+        Get ASTs.
+        {module2}
+        {module1}
+        Linting 2 modules.
+        tests/regrtest_data/wrong_import_position.py (1 of 2)
```

Comment:
This and line 362 and 370 probably also should be f-strings I think?

After that I think we can remove the calls to `_clean_paths`?

Reply:
Right I was reproducitthe text as I though it was clearer. The path cleaning is likely still required would have to check.

Reply:
Updated.  `_clean_paths` is still required.